### ISSUE TITLE
No definition present for homogenous systems of equations

### ISSUE THREAD

**Issue by raduf** [mml-book/mml-book.github.io#410]
**Is your feature request related to a problem? Please describe.**
This is probably a trivial concept in linear algebra, and may not be of much importance in the context of what the book is teaching, but I was looking for the definition in the book and couldn't find it in the side bar notes or in the text: the definition of a homogeneous system of equations. The term is being referred in the beginning of the book, chapter 2.3.3 where kernel and null space are being introduced. Also, although there are exercises referring the term and also to the oposite of the term: inhomogeneous systems of equations

**Describe the solution you'd like**
A definition of what makes a system of equations homogeneous or not should be added, also a side bar mention of it.

**Describe alternatives you've considered**
The reader can always look it up online for some other resources to explain it

**Additional context**

**Comment by mpd37** [comment]
Agreed: It would have been good to have this somewhere, but we can no longer make these changes.

**Comment by fkdosilovic** [comment]
I would like to point out that the issue is present earlier in the same subsection (_2.3. Solving the System of Linear Equations_). In subsubsection _2.3.1. Particular and General Solution_ you mention both particular and general solution to the equation, but do not mention that _general solution = particular solution + homogeneous solution_, even though you mention "2. Find all solutions to **Ax=0**" and then "3. Combine the solutions from steps 1. and 2. to the general solution."

You mention homogeneous system of equations multiple times (as @raduf said), specifically in:
- _2.3.2. Elementary Transformations_, Example 2.7 (Reduced Row Echelon Form) where at the end of the example you mention "In the end, we are still solving a homogeneous equation system."
- _2.3.3. The Minus-1 Trick_ you write "In the following, we introduce a practical trick for reading out the solutions **x** of a homogeneous system of linear equations **Ax = 0**, ..." and ".. as pivots are solutions of the homogeneous equation system **Ax = 0** ." So it is possible to infer what the homogeneous equation system actually is, but it would be nice if it is mentioned in subsubsection 2.3.1.
